src/components/WalletListItem/WalletListItem.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ *
+ * @flow
+ */
+
+import React, { PureComponent } from 'react'
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import I18n from 'react-native-i18n'
+import PropTypes from 'prop-types'
+// import PrimaryBalanceContainerFactory from '../containers/PrimaryBalanceContainerFactory'
+// import PrimaryTokenContainerFactory from '../containers/PrimaryTokenContainerFactory'
+// import TokensListContainerFactory from '../containers/TokensListContainerFactory'
+import isNumber from '../../common/utils/numeric'
+import {
+  indicator_receiving_0,
+} from '../../images'
+import WalletImage from '../WalletImage'
+import styles from './WalletListItemStyles'
+
+const Transactions = ({ transactions }) => !transactions ? null : (
+  !transactions[1] ? (
+    <Image
+      source={indicator_receiving_0}
+    />
+  ) : (
+      <View style={styles.transactionsNumberContainer}>
+        <Text style={styles.transactionsNumber}>
+          {transactions.length}
+        </Text>
+      </View>
+    )
+)
+
+class TokensCounter extends PureComponent {
```

Comment:
I suggest to move this class into new file 'TokensCounter'. We need to start using "one class - one file" principle.
But this principle only for classes, not for functional components like 'Transactions' above.